tools/webdriver/webdriver/bidi/modules/script.py

```diff
@@ -36,15 +36,14 @@ class Script(BidiModule):
     def evaluate(self,
                  expression: str,
                  target: Target,
-                 await_promise: Optional[bool] = None,
+                 await_promise: bool = True,
```

Comment:
Thanks for updating this module, I forgot to do it when updating the tests.
Shouldn't we align, and make the arg mandatory as well for wdpsec tests as well?
The current version will make it default to true, right?

Reply:
Here is an argument for making arg default: https://github.com/web-platform-tests/wpt/pull/34372#discussion_r905832246

Regarding `script.evaluate` tests: as I understood you suggest to add tests checking the `await_promise` is mandatory. I'd prefer to avoid unnecessary growing of PRs, so let's keep it out of scope of this one.

Reply:
FWIW, I was not suggesting to test that `await_promise` is mandatory, I only meant that it should be mandatory by design :) (ie no default value). But I missed that there was already a thread about this, so let's continue there.